app/controllers/application_controller.rb

```diff
@@ -51,4 +52,15 @@ def complete_request(action: nil, path: nil)
   def authorize!
     raise NoMethodError
   end
+
+  def check_college
```

Comment:
I know this is what we did in reservations but I really don't like it -- especially because it means that we need to create this model in all of our feature specs.

I would rather have some sort of persistent nag / a form on the home page to update these settings until they are set?